bin/oneoff/import_unit_details.rb

```diff
@@ -100,7 +100,10 @@ def main(options)
 
     lesson_pairs.each do |lesson, cb_lesson|
       lesson.update_from_curriculum_builder(cb_lesson)
+      log("update lesson #{lesson.id} with cb lesson data: #{cb_lesson_data.to_json[0, 50]}...")
     end
+    script.lessons.select(&:lockable).each(&:update_from_curriculum_builder)
```

Comment:
Yep. This came up because fixing the level positions would fail on non-CSF courses. Though I didn't realize that some lockable lessons have lesson plans, so I'll need to update the logic here.